content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -231,6 +231,11 @@ curl -L "https://github.com/containernetworking/plugins/releases/download/${CNI_
 
 Define the directory to download command files  
 
+{{< note >}}
+The DOWNLOAD_DIR should point to a writable directory. In case of 
```

Comment:
{{< note >}}
The `DOWNLOAD_DIR` variable must be set to a writable directory.
For the Flatcar Container Linux distribution, set `DOWNLOAD_DIR=/opt/bin`.
{{< /note >}}

Reply:
changed, thanks!